src/prefect/client/secrets.py

```diff
@@ -107,5 +106,5 @@ def get(self) -> Optional[Any]:
                 }
                 """,
                 variables=dict(name=self.name),
-            )  # type: Any
-            return as_nested_dict(result.data.secretValue, dict)
+            )
+            return result.data.secretValue
```

Comment:
Not sure if we have a test for this, but could we make sure that we're testing that `secretValue`, if a pure `dict`, remains a pure `dict`? (i.e., I don't want my secret dictionary to get autoconverted to a Box object)